app/src/main/java/com/sun_asterisk/moviedb_49/data/source/remote/fetchdata/GetDataJson.kt

```diff
@@ -0,0 +1,14 @@
+package com.sun_asterisk.moviedb_49.data.source.remote.fetchdata
+
+import com.sun_asterisk.moviedb_49.data.model.Movie
+import com.sun_asterisk.moviedb_49.data.source.remote.OnFetchDataJsonListener
+import com.sun_asterisk.moviedb_49.utils.Contract
+
+class GetDataJson(private val listener: OnFetchDataJsonListener<Movie>) {
+    fun getDate(movie: String) {
+        val mTopRateUrl =
```

Comment:
```suggestion
        val topRateUrl =
```
Bên kotlin không cần có prefix phía trước